glory wanted a **not outdated** list (#8)
Anastasia-Romanov/lacan-user-guide: economy.md

## bal

> - Test the fucking command and you'll find out glory. (you apparentally are incapable of doing this whenever coding!)

Reviewer: ```suggestion
- Test the command and you'll find out glory. (You're apparently incapable of doing this whenever you work on Lacan!)
```

> - Test the fucking command and you'll find out glory. (you apparentally are incapable of doing this whenever coding!)

Reviewer: If you could add screenshots of the various bugs that would be great.

Author: no

## buy

- Doesn't respoond if you specify an invalid item ID.

> - "daily" and weekly" is definitely a clear interface of what an ID could be

Reviewer: ```suggestion
```

There is nothing on this command that says anything about an ID, nor is it unclear (possibly the only part of Lacan with a clear UI).
